### PR TITLE
LPD-33798 Add null check for 'emails' and 'email' getters

### PR DESCRIPTION
**Pull request by tomibiro**

### CODE REVIEW

**Comment by tomibiro** [comment]
ci:test:relevant

**Comment by alvarosaugarlr** [comment]
ci:test:security

**Comment by alvarosaugarlr** [comment]
Hi @tomibiro ,

I have run ci:test:security too because it is needed to check the relation tests.
It is needed to review the results of the tests.

About the code I agree with with the validations, because  `email`  is the [standard claim of the specification](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims).


Tests are needed.

File: modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/OpenIdConnectAuthenticationHandlerImpl.java [hunk 1]
```diff
@@ -150,7 +150,16 @@ public void processAuthenticationResponse(
 
 			List<String> emails = jwtClaimsSet.getStringListClaim("emails");
 
-			claims.put("email", emails.get(0));
+			if (emails == null || emails.isEmpty()) {
```
Comment (rafaprax) [inline modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/OpenIdConnectAuthenticationHandlerImpl.java:153]: Hey @tomibiro, use the `ListUtil#isEmpty` method instead and add tests to cover the issue. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rafaprax** [comment]
Hey @alvarosaugarlr, once you finish the review, please ping me so I can do the final check? 

Thanks in advance

**Comment by tomibiro** [comment]
ci:test:relevant

**Comment by alvarosaugarlr** [comment]
ci:test:stable

**Comment by alvarosaugarlr** [comment]
ci:test:security

**Comment by alvarosaugarlr** [comment]
ci:test:relevant

**Comment by alvarosaugarlr** [comment]
We are going to modify the claim that is readed, so we have to change the test too. 
When the changes are made, it is needed to verify that this change does not affect to other tests (reviewing the ci results).

**Comment by tomibiro** [comment]
ci:test:relevant

**Comment by tomibiro** [comment]
Closing this and opening a new one for the clean sight: https://github.com/liferay-appsec/liferay-portal/pull/1924

**Pull request closed without merging**